tethys_portal/views/email.py

```diff
@@ -0,0 +1,17 @@
+from django.conf import settings
+from django.contrib.auth.views import PasswordResetView
+
+
+class TethysPasswordResetView(PasswordResetView):
```

Comment:
Please add tests for this class to get coverage back to 100%.

Reply:
@swainn Would you be opposed to me just switching the angle bracket checks to ternary operators? It would result in the same functionality and 100% coverage.

Reply:
While that satisfies the 100% coverage need, it does not satisfy the need to test the two cases (with and without angle brackets) and make sure the implementation works. The test file structure mirrors the file structure of the projects and there are ample examples of tests throughout. Django has a decorator that can be used to override settings values in tests.

Perhaps I can help you with this on Tethys Scrum tomorrow?

Reply:
I kind of figured you'd say that :) If you email me the link/invitation to Tethys scrum, then I will join and we can work on it then. I'll make sure to have everything else completed beforehand. Thanks.